google-http-client/src/main/java/com/google/api/client/http/javanet/NetHttpRequest.java

```diff
@@ -74,31 +74,17 @@ public LowLevelHttpResponse execute() throws IOException {
         } else {
           connection.setChunkedStreamingMode(0);
         }
-        OutputStream out = connection.getOutputStream();
-        boolean threw = true;
-        try {
+
+        try (OutputStream out = connection.getOutputStream()) {
```

Comment:
Yeah... We don't do that in this code base, because of the Java version restriction.

Reply:
Ok, good context. Thank you.